sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -302,7 +302,9 @@ private static void ensureAttachmentSizeLimit(
             SentryItemType.ProfileChunk,
             () -> cachedItem.getBytes().length,
             "application-json",
-            traceFile.getName());
+            traceFile.getName(),
+            null,
+            "android");
```

Comment:
I'm not sure if/how hybrid SDKs are using this, but should this always be "android"?

Reply:
The platform in the chunk item-type header should always match the platform in the chunk item-type payload. 

So maybe instead of hard coding `android`, using `profileChunk.getPlatform()` could be better ?

Reply:
@markushi i did it this way so the other item headers are not changed. Also, the same code creates and sends the envelopes of the jvm world, so it cannot be set always to "android"
Regarding hybrid, we will see, but they shouldn't use our `captureProfileChunk` method, so it should be fine
@viglia 👍